schema/sale_artworks.js

```diff
@@ -7,9 +7,7 @@ import { SaleArtworkAggregations, SaleArtworkCounts } from "schema/filter_sale_a
 
 const DEFAULTS = {
   aggregations: ["total"],
-  page: 1,
-  size: 10,
-  offset: 0,
+  first: 10,
```

Comment:
Cool. Nice work on the default for this, I never remember `first` and always spend some minutes wondering why the connection returned an empty list.

Reply:
good call, maybe we should do that in all connections?